src/renderer/base/renderer.cpp

```diff
@@ -611,14 +588,13 @@ bool Renderer::IsGlyphWideByFont(const std::wstring_view glyph)
     //      renderer. We won't know which is which, so iterate over them.
     //      Only return the result of the successful one if it's not S_FALSE (which is the VT renderer)
     // TODO: 14560740 - The Window might be able to get at this info in a more sane manner
-    FAIL_FAST_IF(!(_rgpEngines.size() <= 2));
-    for (IRenderEngine* const pEngine : _rgpEngines)
+    foreachEngine(pEngine)
     {
         const HRESULT hr = LOG_IF_FAILED(pEngine->IsGlyphWideByFont(glyph, &fIsFullWidth));
         // We're looking for specifically S_OK, S_FALSE is not good enough.
         if (hr == S_OK)
         {
-            return fIsFullWidth;
+            break;

```

Comment:
Does this actually make a difference in the final asm code? I assume the compiler is smart enough to know this is the exact same thing. But I don't really know nothing about compilers. Just my guess.

Reply:
I doubt it makes a difference... I just found it to be redundant.